app/avo/resources/user.rb

```diff
@@ -33,7 +33,6 @@ def fields # rubocop:disable Metrics
     field :email_reset, as: :boolean
     field :handle, as: :text
     field :public_email, as: :boolean
-    field :twitter_username, as: :text, as_html: true, format_using: -> { link_to value, "https://twitter.com/#{value}", target: :_blank, rel: :noopener if value.present? }
```

Comment:
Can you rename to `social_link` (and follow the same for helpers ...) and remove the forma_using here (swap with validation on model side to be valid link)?